### DISCUSSION TITLE
Exchanges to Support

### DISCUSSION THREAD

**Discussion by bmoscon** [bmoscon/cryptofeed#368] [Ideas]
A list of exchanges that support websockets:

[Bithumb](https://apidocs.bithumb.com/docs/websocket_public)

[KuCoin](https://docs.kucoin.com/#websocket-feed)

[Liquid](https://developers.liquid.com/#iii.-liquid-tap-websocket)

[Delta](https://docs.delta.exchange/#websocket-feed)

[Crypto.com](https://exchange-docs.crypto.com/#websocket-root-endpoints)

I'm sure there are many others that could be supported - if you know of one you'd like supported, leave a comment below with the name of the exchange and a link to the websocket api docs

**Comment by 0trade** [comment]
That is great news, especially in KuCoin.

BitBay is look like good, The api doc is [here.](https://docs.bitbay.net/v1.0.1-en/reference#public-subscription)
